config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -271,6 +271,6 @@ periodics:
   annotations:
     testgrid-num-failures-to-alert: '2'
     testgrid-alert-stale-results-hours: '48'
-    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind
+    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind, sig-release-master-blocking
```

Comment:
```
    config_test.go:480: dashboard_tab sig-release-master-blocking/conformance-ga-only is release-blocking: - Must have alert_options
```

Reply:
I thought it did...

```
  annotations:
    testgrid-num-failures-to-alert: '2'
    testgrid-alert-stale-results-hours: '48'
    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind, sig-release-master-blocking
    testgrid-tab-name: conformance-ga-only
    testgrid-alert-email: liggitt@google.com

```

Reply:
I think what it's missing actually is a description.

Reply:
oh, ok... added a description